packages/insomnia/src/ui/components/key-value-editor/key-value-editor.tsx

```diff
@@ -73,7 +73,9 @@ export const KeyValueEditor: FC<Props> = ({
   valuePlaceholder,
   readOnlyPairs,
 }) => {
-  const [showDescription, setShowDescription] = React.useState(false);
+  const [showDescription, setShowDescription] = React.useState(
```

Comment:
Nitpick since we import useMemo from react maybe we should do the same for useState
```suggestion
  const [showDescription, setShowDescription] = useState(
```